### PR TITLE
update EasyBuild bootstrap script to download distribute tarball from http://easybuilders.github.io/easybuild/files

### PR DESCRIPTION
**Pull request by boegel**
The download of the `distribute` tarball is going to break when https://github.com/hpcugent/easybuild is transferred to https://github.com/easybuilders/easybuild later today.

This can be fixed by including `distribute-0.6.49-patched1.tar.gz` in https://github.com/hpcugent/hpcugent.github.com/pull/3 (which I'll probably do), but the bootstrap script should be updated accordingly.

### CODE REVIEW

**Comment by boegel** [comment]
(close/open to trigger Travis)

**Comment by boegel** [comment]
@easybuild-framework-maintainers please review?

File: .travis.yml [hunk 1]
```diff
@@ -88,7 +88,7 @@ script:
     - EB_BOOTSTRAP_VERSION=$(grep '^EB_BOOTSTRAP_VERSION' $TRAVIS_BUILD_DIR/easybuild/scripts/bootstrap_eb.py | sed 's/[^0-9.]//g')
     - EB_BOOTSTRAP_MD5SUM=$(md5sum $TRAVIS_BUILD_DIR/easybuild/scripts/bootstrap_eb.py | cut -f1 -d' ')
     - EB_BOOTSTRAP_FOUND="$EB_BOOTSTRAP_VERSION $EB_BOOTSTRAP_MD5SUM"
-    - EB_BOOTSTRAP_EXPECTED="20170503.01 fdf8ca02b6600c4ce0b6a02b88f1f40a"
+    - EB_BOOTSTRAP_EXPECTED="20170705.01 90d2dd008be027c35c1d194cbf5b1d44"
```
Comment (damianam) [inline .travis.yml]: Can this hash be sha256? If so, I'd say this is the perfect moment for it.

Comment (boegel) [inline .travis.yml]: Sure. I don't think it matters much, this is just a mechanism just help with not forgetting to bump the version of the bootstrap script (as opposed to actually verifying a checksum), but it doesn't hurt either. Done in d28755d7.

**Comment by damianam** [comment]
lgtm

**Comment by boegel** [comment]
broken tests are fixed in #2258, let's get that one merged first...

**Comment by boegel** [comment]
@easybuilders/easybuild-framework-maintainers Good to go!

File: .travis.yml [hunk 2]
```diff
@@ -86,9 +86,9 @@ script:
     # check bootstrap script version
     # version and MD5 are hardcoded below to avoid forgetting to update the version in the script along with contents
     - EB_BOOTSTRAP_VERSION=$(grep '^EB_BOOTSTRAP_VERSION' $TRAVIS_BUILD_DIR/easybuild/scripts/bootstrap_eb.py | sed 's/[^0-9.]//g')
-    - EB_BOOTSTRAP_MD5SUM=$(md5sum $TRAVIS_BUILD_DIR/easybuild/scripts/bootstrap_eb.py | cut -f1 -d' ')
+    - EB_BOOTSTRAP_MD5SUM=$(sha256sum $TRAVIS_BUILD_DIR/easybuild/scripts/bootstrap_eb.py | cut -f1 -d' ')
```
Comment (damianam) [inline .travis.yml]: Ehmmmm, is it just me or having a sha256 hash in a variable called `SOMETHING_SOMETHING_MD5SUM` is a bad idea?

Comment (boegel) [inline .travis.yml]: It's just you...

I mean: fixed in 83b41802 ;)

**Comment by damianam** [comment]
lgtm. In it goes!

**Pull request merged**